src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -340,22 +349,29 @@ private void testEditInstructorAction() throws Exception {
         courseEditPage.selectRoleForInstructor(editInstructorIndex, "Custom");
         
         // deselect some privileges from Co-owner default values
-        courseEditPage.clickCourseLevelPrivilegesLink(editInstructorIndex, 0);
-        courseEditPage.clickCourseLevelPrivilegesLink(editInstructorIndex, 1);
-        courseEditPage.clickCourseLevelPrivilegesLink(editInstructorIndex, 7);
+        courseEditPage.clickCourseLevelPrivilegesLink(editInstructorIndex,
+                InstructorCourseEditPage.COURSE_MODIFY_COURSE);
+        courseEditPage.clickCourseLevelPrivilegesLink(editInstructorIndex,
+                InstructorCourseEditPage.COURSE_MODIFY_INSTRUCTORS);
+        courseEditPage.clickCourseLevelPrivilegesLink(editInstructorIndex,
+                InstructorCourseEditPage.COURSE_MODIFY_OTHERS_COMMENTS);
         courseEditPage.clickAddSectionLevelPrivilegesLink(editInstructorIndex);
         courseEditPage.clickSectionSelectionCheckBox(editInstructorIndex, 0, 0);
         courseEditPage.clickSectionSelectionCheckBox(editInstructorIndex, 0, 1);
```

Comment:
The meaning of the line above (line 360 in this case) would be "for the instructor at `editInstructorIndex`, for the 1st section panel, click on the 2nd section".

It would be a bit difficult to refactor some of these numbers out as:
1. The number is not tied to any other meaning.
2. The number is not used frequently enough to warrant using a variable for it.